### PR TITLE
Add feedback snippet

### PR DESCRIPTION
**Pull request by KavithaSiva**
## What Has Changed?

I have added feedback snippets to all pages in our documentation portal. Currently, I have explicitly added the snippet to each page, in a follow up PR, I will try to add the snippet in a configuration file centrally so that it gets added to all pages automatically.

Mainly look at files: src/sap/common-utils/DocsRating/docRating.css and src/sap/common-utils/DocsRating/index.js as these files contain the code for the feedback snippet, all other changes are just adding the component to the file.

## Manual Checks?

- [x] Text adheres to the style guide (`vale docs/`)
  - Every sentence is on its own line
  - Headings use [title capitalization](https://capitalizemytitle.com/style/AP/#) (applies also to the sidebar and title of a document)
  - You followed [naming center](https://www.sapbrandtools.com/naming-center/#/dashboard) guidelines when referring to SAP products (e.g. SAP S/4HANA)
- [x] You checked your spelling and grammar (consider using Grammarly, see `CONTRIBUTING.md`)
- [x] You formatted all changed files with prettier (`npm run prettier`)
- [x] You tested if the documentation still builds (`npm run build`)
- ~[x ] You verified all new and changed links still work (changing the `id` or name of a file can break links)~
- ~[x]You have updated the [feature matrix](https://github.com/SAP/cloud-sdk/blob/main/docs/components/data/features.js) if you add documentation on a new feature or otherwise required.~

### CODE REVIEW

File: src/sap/common-utils/DocsRating/docRating.css [hunk 1]
```diff
@@ -0,0 +1,35 @@
+.docsRating svg{
+    height: 1.5em;
+    margin: 0 0 0 6px;
+    width: 1.5em;
+    cursor:pointer;
+
+}
```
Comment (Johannes-Schneider) [inline src/sap/common-utils/DocsRating/docRating.css:7]:
```suggestion
}

```

File: src/sap/common-utils/DocsRating/index.js [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+
+const DocsRating = ({ label }) => {
+
+  const [haveVoted, setHaveVoted] = useState(false);
+  const giveFeedback = value => {
+
+   if (swa && swa.hasOwnProperty("trackCustomEvent")) {
+     swa.trackCustomEvent("collectFeedback", value);
+     }
+     setHaveVoted(true);
```
Comment (Johannes-Schneider) [inline src/sap/common-utils/DocsRating/index.js:14]: Maybe this just GitHub, but indentation seems to be off. Please check in your local IDE.

**Review by Johannes-Schneider — COMMENTED** [review]
Left some comments with regard to code formatting.
Unfortunately, I can't say too much about the actual content.

**Comment by KavithaSiva** [comment]
> Left some comments with regard to code formatting.
> Unfortunately, I can't say too much about the actual content.

You can perhaps run the changes locally and see if you like the color scheme and the style of font/image used in the snippet. Otherwise thanks for the other comments.

**Review by artemkovalyov — APPROVED** [review]
The update looks fine. If manually tested locally - it's good to go.

File: src/sap/common-utils/DocsRating/index.js (hunk 2 above)
Comment (KavithaSiva) [reply src/sap/common-utils/DocsRating/index.js]: Done.

**Pull request merged**